<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BNGECO-L-89144" name="firefox" tests="2" failures="2" timestamp="27 Jun 2019 14:46:23 GMT" time="0.349" errors="0">
  <testcase name="verifyFlightBooking" time="0.23" classname="com.pack.common.tests.FlightConfirmationPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.pack.functionlibrary.FunctionLibrary.setValue(FunctionLibrary.java:133)
	at com.pack.common.pageobjects.HomePage.enterUserName(HomePage.java:35)
	at com.pack.common.tests.FlightConfirmationPageTest.verifyFlightBooking(FlightConfirmationPageTest.java:32)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- verifyFlightBooking -->
  <testcase name="verifyFlightBooking" time="0.34" classname="com.pack.common.tests.FlightConfirmationPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.pack.utils.Listeners.TestListener.takeScreenShot(TestListener.java:96)
	at com.pack.utils.Listeners.TestListener.onTestFailure(TestListener.java:67)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- verifyFlightBooking -->
</testsuite> <!-- firefox -->
